index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BOSE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
```

Comment:
You should be running the application using `npm start` so in this case you should link to scss files, as the project is configured to process them. Remove style.css and style.css.map

Reply:
Hi, i have a problem with this. I move all files to src and in index.html link to main.scss file and i have error, look at my attachment
![error](https://github.com/mate-academy/layout_miami/assets/125818094/4013d0eb-0029-442f-b791-b40ce3c22416)

Reply:
What is wrong?